### PR TITLE
Upgrade packages to address Vanta vulnerabilities

### PR DESCRIPTION
**Pull request by iwong3**
## Description of the change

Addresses this vulnerability:
image

Also bump a few other packages to match outline2.0

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -5,9 +5,9 @@ go 1.18
 require (
 	github.com/aws/aws-sdk-go v1.44.38
 	github.com/pkg/errors v0.9.1
-	github.com/sirupsen/logrus v1.8.1
+	github.com/sirupsen/logrus v1.9.3
```
Comment (earlofhemsley) [inline go.mod:8]: i am pretty sure you need to run `go install -v -mod vendor .` to get all changes mirrored down into the `vendor` folder. Or, are we moving away from vendoring?

Comment (iwong3) [reply go.mod:8]: Hmm, I noticed that too - `outline2.0` has a `vendor` folder but we don't here. I can run that.

Comment (earlofhemsley) [reply go.mod:8]: I'm not sure if we want/need to if it's not already there. I know that `vendor` isn't totally necessary and may not even be required, but I'm not knowledgable enough on the subject to know. Maybe @james-m does?

File: go.sum [hunk 2]
```diff
@@ -1,795 +1,44 @@
-cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```
Comment (iwong3) [inline go.sum:1]: I think all these changes are because we never ran `go mod tidy` until just now.

File: go.mod (hunk 1 above)
Comment (iwong3) [reply go.mod:8]: I'm guessing we don't need it as we don't have a vendor folder here. Everything still builds and runs fine without it. Not sure on the details either though.

Comment (iwong3) [reply go.mod:8]: image
Holy cow

Comment (earlofhemsley) [reply go.mod:8]: yeah, it's for that exact reason that we don't want it if we don't need it.

Comment (gzphreak) [reply go.mod:8]: I don't want us to check in the vendor folder. I'd prefer we don't check it in anywhere ... in any repo.

**Review by earlofhemsley — APPROVED** [review]
LGTM. Will there be a companion PR at some point to replace all usages of `env-utils` in `outline2.0` with this specific version? That would be really helpful, I think.

**Comment by iwong3** [comment]
> LGTM. Will there be a companion PR at some point to replace all usages of `env-utils` in `outline2.0` with this specific version? That would be really helpful, I think.

Yep, I'll work on that as part of my PR to resolve the Vanta vulnerabilities.

**Pull request merged**